test/dotcom_web/channels/predictions_channel_test.exs

```diff
@@ -86,6 +86,31 @@ defmodule DotcomWeb.PredictionsChannelTest do
       # Verify
       assert_push("data", %{predictions: ^predictions})
     end
+
+    test "filters out past predictions", context do
+      # Setup
+      now = Timex.now() |> Timex.shift(seconds: 1)
+      past = Timex.shift(now, seconds: -15)
+      future = Timex.shift(now, seconds: 15)
+
+      predictions =
+        [past, now, future]
+        |> Enum.map(&Prediction.build(:canonical_prediction, %{departure_time: &1}))
+
+      expect(@predictions_pub_sub, :subscribe, fn _ ->
+        predictions
+      end)
+
+      {:ok, _, socket} = subscribe_and_join(context.socket, PredictionsChannel, context.channel)
+
+      # Exercise
+      PredictionsChannel.handle_info({:new_predictions, predictions}, socket)
+
+      [_past_prediction | expected_predictions] = predictions
```

Comment:
```suggestion
      expected_predictions = [now, future]
```

Not a huge deal, but I'm more in favor of being super explicit about exactly what we expect, rather than having to reason about what's in the `predictions` array.